cwl/src/main/scala/wdl4s/cwl/Cwl.scala

```diff
@@ -12,15 +14,15 @@ sealed trait Cwl {
 
 case class Workflow(
   cwlVersion: Option[CwlVersion] = None,
-  `class`: Witness.`"Workflow"`.T,
+  `class`: Witness.`"Workflow"`.T = "Workflow".narrow,
   inputs: WorkflowInput,
   outputs: WorkflowOutput,
   steps: WorkflowSteps) extends Cwl
 
 /**
   *
   * @param inputs
-  * @param outputs
+  * @param outputs There is no logical default for outputs as a job without outputs is not considered useful.
```

Comment:
I'm pretty sure the `1st-tool.cwl` has an empty outputs list.

Useful? No... But valid...? I guess so.